src/portfft/dispatcher/subgroup_dispatcher.hpp

```diff
@@ -134,7 +133,7 @@ PORTFFT_INLINE void subgroup_impl(const T* input, T* output, const T* input_imag
   Idx n_cplx_per_sg = n_ffts_per_sg * fft_size;
   Idx n_reals_per_sg = n_ffts_per_sg * n_reals_per_fft;
   // id_of_fft_in_sg must be < n_ffts_per_sg. Subtraction ensure this for trailing work-items.
-  Idx id_of_fft_in_sg = subgroup_local_id / factor_sg - (subgroup_local_id >= factor_sg * n_ffts_per_sg);
+  Idx id_of_fft_in_sg = std::min(subgroup_local_id / factor_sg, n_ffts_per_sg - 1);
```

Comment:
I think this change makes the comment above redundant. Remove it (or update if you disagree).

Reply:
Addressed in https://github.com/codeplaysoftware/portFFT/pull/142/commits/ca5742962ed755442d1e2af7b39c0f7876386ab5